### ISSUE TITLE
AppiumDriver.findElements() arbitrarily not able to find element for accessibility id

### ISSUE THREAD

**Issue by indramohan92** [appium/appium#8579]
## The problem
AppiumDriver sometimes does not able to find the element by accessibility id even though accessibility id is present in the page. 

## Environment
* Appium version (or git revision) that exhibits the issue: 1.6.3
* Last Appium version that did not exhibit the issue (if applicable): 1.6.2
* Mac OS/version used to run Appium: 10.12.5
* iOS Version under test: 10.2
* Real device or emulator/simulator: iPad Air 2
* Xcode Version: 8.2.1 

## Details
Please refer the line number 2857 in the logs. Here AppiumDriver try the find element with accessibility id "YES, I AM READY" but failed to get the snapshots, but it is able the access same element with same accessibility id in the other test cases. Refer line numbers 369, 402, 435 in the logs. 

## Link to Appium logs
https://gist.github.com/ADYASHA-PADHI/047f8f3063b2caabc0e42912af81f9b2

**Comment by mykola-mokhnach** [comment]
The request crashed WDA. Try to update Appium to 1.6.5
Report the issue directly to https://github.com/facebook/WebDriverAgent/issues if it still does not work, since it has nothing to do with Appium
